fhirzeug/generators/python_pydantic/static_files/tests/test_examples/test_import_examples.py

```diff
@@ -93,6 +77,29 @@ def test_read_write(fhir_file: Path):
     assert counter_in == counter_out
 
 
+def test_primitive_extension_exists(fhir_file: Path):
+    """Test each primitive field has the possibility of an extension.
+
+    If a field type is forgotten in the generator settings (under
+    mapping_rules > jsonmap), this test might be able to spot it.
+    Note: if no example uses this field, then no errors will be raised.
+    """
+    with _open_file(fhir_file) as f_in:
+        doc = json.load(f_in)
+
+    resource = r4.from_dict(doc)
+    for field, value in resource:
+        _check_field_extension_existence(resource, field, value)
+
+
+def _open_file(fhir_file: Path) -> _io.TextIOWrapper:
```

Comment:
Related to the comment above.

```suggestion
def _open_file(fhir_file: Path) -> io.TextIOWrapper:
```